### ISSUE TITLE
git ftp download does not work with FTPES

### ISSUE THREAD

**Issue by tortlepp** [git-ftp/git-ftp#343]
Hi, today I tried git-ftp and found a problem:  I set up git-ftp with `git config git-ftp.url ftpes://ftp.example.com/something` and `git ftp push` worked well. But when I try `git ftp download`, it fails with the message `lftp: ftpes - not supported protocol`.
But LFTP supports FTPES, you can configure it e.g. in _/etc/lftp.conf_ . But to use FTPES in LFTP, the protocol has to be _ftp://_ instead of _ftpes://_ (see http://linux.overshoot.tv/wiki/networking/lftp#A_multitude_of_protocols). I guess that git-ftp does not respect this fact.

**Comment by mkllnk** [comment]
True, Git-ftp just passes the protocol on to lftp. Thank you for reporting that. The fix shouldn't be hard. We'll see if someone has time. :-)
